plutus-core/plutus-core/src/PlutusCore/Builtin/Runtime.hs

```diff
@@ -59,11 +67,16 @@ data BuiltinRuntime val =
                               -- can put @undefined@ here. TODO: we should test if making this
                               -- strict introduces any measurable speedup.
 
+instance NFData (BuiltinRuntime val) where
+    rnf (BuiltinRuntime rs f exF) = seq rs $ seq f $ rwhnf exF
```

Comment:
Shouldn't these all be using `deepseq`?

Reply:
i forgot to rnf the `rs` indeed. the rest don't matter

Reply:
Why not?

Reply:
The f and exf are functions so their `nfdata.rnf === rwhnf` which is the same as `seq`ing them

Reply:
But I could stick to the normal path and rnf those f and exf as well

Reply:
Easier and safer! I think they are actually type families so they might not be functions!